crates/ruff_python_ast/src/helpers.rs

```diff
@@ -1746,4 +1774,67 @@ y = 2
 
         Ok(())
     }
+
+
+    #[test]
+    fn any_over_type_param_type_var() {
+        let type_var_no_bound = TypeParam::TypeVar(TypeParamTypeVar {
+            range: TextRange::new(TextSize::from(0), TextSize::from(0)),
+            bound: None,
+            name: Identifier::new("x", TextRange::new(TextSize::from(0), TextSize::from(0))),
+        });
+        assert_eq!(
```

Comment:
You can also use `assert!` if comparing to a boolean, but I don't feel strongly that it's "better".